pkg/netutil/netutil.go

```diff
@@ -475,7 +475,7 @@ func (e *CNIEnv) parseSubnet(subnetStr string) (*net.IPNet, error) {
 		return nil, err
 	}
 	if subnetStr == "" {
-		_, defaultSubnet, _ := net.ParseCIDR(DefaultCIDR)
+		_, defaultSubnet, _ := net.ParseCIDR(StartingCIDR)
```

Comment:
Need to define `StartingCIDR` in https://github.com/containerd/nerdctl/blob/main/pkg/netutil/netutil_windows.go

Reply:
Fixed. Thank you.